vertical-pod-autoscaler/pkg/recommender/model/vpa.go

```diff
@@ -127,11 +127,25 @@ func (vpa *Vpa) UseAggregationIfMatching(aggregationKey AggregateStateKey, aggre
 	}
 	if vpa.matchesAggregation(aggregationKey) {
 		vpa.aggregateContainerStates[aggregationKey] = aggregation
+		aggregation.IsUnderVPA = true
 		return true
 	}
 	return false
```

Comment:
I think we should set IsUnderVPA = false here in case someone deleted VPA object.

Reply:
Actually I added this to Delete VPA in cluster.go